public/app/ui/pages/home/home.html

```diff
@@ -3,7 +3,7 @@
     <p>
         This website provides access to the Royal Western Australian Historical Society's collection for research and public interest purposes. Searches can be performed on the <a href="/photographs/search">photographic</a> and <a href="/museum/search">museum</a> collections, as well as <a href="/memorials/search">memorial records</a> and the <a href="/library/search">library catalogue</a>.
     </p>
-    <div class="row homepage">
+    <div class="row">
```

Comment:
This class is now unnecessary since we're using `article#home` as the style prefix.